src/appengine/libs/issue_filer.py

```diff
@@ -222,15 +173,29 @@ def file_issue(testcase,
   if testcase.uploader_email and testcase.uploader_email not in ccs:
     ccs.append(testcase.uploader_email)
 
-  if issue_tracker.project == 'oss-fuzz' and ccs:
-    # Add a reported label for deadline tracking.
-    issue.labels.add(reported_label())
+  ccs.extend(properties.ccs)
+
+  # Get view restriction rules for the job.
+  issue_restrictions = data_handler.get_value_from_job_definition(
+      testcase.job_type, 'ISSUE_VIEW_RESTRICTIONS', 'security')
+  should_restrict_issue = (
+      issue_restrictions == 'all' or
+      (issue_restrictions == 'security' and testcase.security_flag))
+
+  if should_restrict_issue:
+    issue.labels.add(policy.label('restrict_view'))
+
+  for label in properties.labels:
+    for result in apply_substitutions(label, testcase, security_severity):
+      if result.startswith(policy.label('reported_prefix')) and not ccs:
```

Comment:
i think use of ccs here in 190 and line 197 should go in a local like
has_accountable_people = bool(ccs) or some other better name to tell purpose of ccs. i thought of even adding issue.owner in the condition, but new bugs we dont autoassign.